app/v1/models/des/ListBFLossesResponse.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.des
+
+import play.api.libs.json.{JsPath, Reads}
+import v1.models.domain.{BFLossId, BFLosses}
+
+case class ListBFLossesResponse(ids: Seq[String]) {
+  def toMtd: BFLosses = BFLosses(ids.map(BFLossId(_)))
```

Comment:
Could this not be replaced with a custom reads directly into the model type? It would just avoid having to call this method manually when it needs to be used.

Reply:
Done